cmd/oci-runtime-tool/runtime_validate.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/mrunalp/fileutils"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/satori/go.uuid"
+	"github.com/urfave/cli"
+)
+
+var runtimeValidateFlags = []cli.Flag{
+	cli.StringFlag{Name: "runtime", Value: "runc", Usage: "OCI runtime"},
+}
+
+var runtimeValidateCommand = cli.Command{
+	Name:   "runtime-validate",
+	Usage:  "validate an OCI runtime",
+	Flags:  runtimeValidateFlags,
+	Before: before,
+	Action: func(context *cli.Context) error {
+		return runtimeValidate(context.String("runtime"))
+	},
+}
+
+func runtimeValidate(runtime string) error {
+	fmt.Println("-----------------------------------------------------------------------------------")
+	fmt.Printf("                  VALIDATING RUNTIME: %q\n", runtime)
+	fmt.Println("-----------------------------------------------------------------------------------")
```

Comment:
I don't think this header is really necessary ;).  Certainly not without a way to turn it off.  But if removing it is part of post-PR polishing, then whatever, we can keep it for now.